src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -107,26 +103,8 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
             final SqlRequest sqlRequest = SqlRequestFactory.getSqlRequest(request);
             LOG.info("[{}] Incoming request {}: {}", LogUtils.getRequestId(), request.uri(), sqlRequest.getSql());
 
-            if (explainRequiresBlockingIOCall(sqlRequest)) {
-                return channel -> {
-                    final Runnable requestExecution = () -> {
-                        try {
-                            final QueryAction action = explainRequest(client, sqlRequest);
-                            executeSqlRequest(request, action, client, channel);
-                        } catch (Exception e) {
-                            logAndPublishMetrics(e);
-                            reportError(channel, e, isClientError(e) ? BAD_REQUEST : SERVICE_UNAVAILABLE);
-                        }
-                    };
-
-                    final ThreadPool threadPool = client.threadPool();
-                    threadPool.schedule(threadPool.preserveContext(requestExecution), new TimeValue(0L),
-                            AsyncRestExecutor.SQL_WORKER_THREAD_POOL_NAME);
-                };
-            } else {
-                final QueryAction queryAction = explainRequest(client, sqlRequest);
-                return channel -> executeSqlRequest(request, queryAction, client, channel);
-            }
+            final QueryAction queryAction = explainRequest(client, sqlRequest);
```

Comment:
That change doesn't seem to relate to the rest of the PR, so: why did all the code on the left go?